src/components/incidents/Incidents.js

```diff
@@ -134,6 +138,9 @@ const Incidents = () => {
       });
     }
     setData(falsiesRemoved(filtered));
+    setAdded([]); // it cleans checked data when we change the filtered data
+    setSelectedIncidents([]);
+    // console.log('sam', sam);
```

Comment:
We should probably get rid of this console.log if we don't need it. Comments on code a clear and descriptive of what it does. Good job!